keyboards/exclusive/e6_rgb/e6_rgb.c

```diff
@@ -190,3 +176,30 @@ led_config_t g_led_config = { {
 } };
 
 #endif
+
+bool process_record_kb(uint16_t keycode, keyrecord_t *record) {
+    if (record->event.pressed) {
+        switch(keycode) {
+#if defined(RGBLIGHT_ENABLE) && defined(RGB_MATRIX_ENABLE)
```

Comment:
This doesn't seem to make sense, currently only one of these can be enabled at once.

Reply:
actually the underglow leds and per key rgb led can turn on at the same time.

Reply:
These switch cases only call RGB matrix functions, so the `RGBLIGHT_ENABLE` check is entirely unnecessary. I think that enabling both may cause some weird behaviour, as the `RGB_*` keycodes are shared. Thus, it might be *technically* possible, but I suspect won't behave correctly right now.

Reply:
Actually, it looks like the `rgblight_*` functions, called by the `RGB_*` keycodes, are remapped to the RGB matrix functions if Underglow is disabled. So...it looks like you are correct, and I understand this switch block is adding RGB matrix keycodes since with both enabled the core keycodes will only change the underglow LEDs.

Reply:
yes, because currently QMK did not have rgb matrix specific keycodes, so I just select F13,F14 for using with qmk configurator.